packages/client-sdk-web/src/internal/pubsub-client.ts

```diff
@@ -134,6 +134,15 @@ export class PubsubClient<
     request.setResumeAtTopicSequenceNumber(
       options.subscriptionState.resumeAtTopicSequenceNumber
     );
+    if (options.subscriptionState.lastTopicSequencePage !== undefined) {
+      request.setSequencePage(options.subscriptionState.lastTopicSequencePage);
+    }
+
+    this.getLogger().trace(
+      'Subscribing to topic with resume_at_topic_sequence_number %s and sequence_page %s',
+      options.subscriptionState.resumeAtTopicSequenceNumber,
+      options.subscriptionState.lastTopicSequencePage ?? 'undefined'
+    );
```

Comment:
I recommend adding a property accessor `resumeAtTopicSequencePage` on subscription state class which defaults to 0. That way we can avoid the null checks here and throughout (see `public get resumeAtTopicSequenceNumber` for reference.)